generalhw/hpt630v1/README.md

```diff
@@ -5,8 +5,8 @@ For details about how this setup works, see
 
 ### Start the job
 
-While logged into openQA server (unless you've installed and configured
-`openqa-cli` locally):
+Before installation, set booting from installer as a default option (or at least
+after the currently installed OS). As of now, this has to be performed manually.
```

Comment:
Would be better to replace `now` with the current date and explain why this isn't automated.

Reply:
https://github.com/3mdeb/openqa-tests-qubesos/compare/c91f3f4dccd0ec59c0a56ad523bb3f9430a1a6df..fee8141d25c417c5b87babaa21c665c0f035762f, hopefully will be added soon.